### ISSUE TITLE
Uploading new TLEs

### ISSUE THREAD

**Issue by burnsfisher** [ac2cz/FoxTelem#772]
For V1.13, I'd recommend adding a menu item to refresh the TLEs.  Currently, this will be mostly useful right after launch when adding a new TLE.  However future satellites could have maneuvering capability and current ones change quickly as they approach re-entry.  

This occurred to me as I was thinking about telling worldwide listeners how to get the new TLEs for MESAT1 right after launch.

**Comment by ac2cz** [comment]
Agreed.  This is not easy to do manually for launches.  It is setup to work in steady state and to not download the TLE file too many times from amsat.org.

**Comment by burnsfisher** [comment]
Appears to work fine.  Thanks!
